inception-pdf-editor/src/main/js/pdfanno/pdfjs/pdf.js

```diff
@@ -4216,139 +4216,95 @@ var SVGGraphics = (function SVGGraphicsClosure() {
         var fnId = opTree[x].fnId;
         var args = opTree[x].args;
 
-        switch (fnId | 0) {
```

Comment:
Why are switch..case statements bad? You need to motivate the change...

Reply:
Changes are updated.